src/main/java/me/redcarlos/higtools/modules/hud/TextPresets.java

```diff
@@ -12,15 +12,15 @@ private static TextHud create() {
     }
 
     static {
-        addPreset("Player-Coords", "XYZ: #1{roundToString(player.pos.x, 1)}, {floor(player.pos.y)}, {roundToString(player.pos.z, 1)}", 0);
-        addPreset("Opposite-Coords", "{player.opposite_dimension != \"End\" ? player.opposite_dimension + \":\" : \"\"} #1{player.opposite_dimension != \"End\" ? \"\" + floor(player.opposite_dim_pos.x) + \", \" + floor(player.opposite_dim_pos.z) : \"\"}", 0);
```

Comment:
Add those back, they use a different rounding method than meteor's that make them more accurate especially for highway stuff